tools/packaging/guest-image/build_image.sh

```diff
@@ -48,7 +48,13 @@ build_initrd() {
 		PULL_TYPE="${PULL_TYPE:-default}" \
 		COCO_GUEST_COMPONENTS_TARBALL="${COCO_GUEST_COMPONENTS_TARBALL:-}" \
 		PAUSE_IMAGE_TARBALL="${PAUSE_IMAGE_TARBALL:-}"
-	mv "kata-containers-initrd.img" "${install_dir}/${artifact_name}"
+
+	if [[ "$artifact_name" == *"nvidia-gpu"* ]]; then
+		nvidia_driver_version=$(cat "${builddir}"/initrd-image/*/nvidia_driver_version)
```

Comment:
I was wondering where `nvidia_driver_version` is created. Thanks!

Reply:
In the rootfs build, I need to follow up with another ton of PRs so all of this makes sense. I hadn't had any reviews ... :)

Reply:
Hi @zvonkok !

Why don't you have the caller to pass the driver version via `--image-initrd-suffix` option? This way we avoid special treatments, keeping this script generic.

Reply:
Because we do not know upfront what the latest driver version is in the Ubuntu repos.

Reply:
Got it!